common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -512,6 +512,16 @@ class CelebornConf(loadDefaults: Boolean) extends Cloneable with Logging with Se
   def pushLimitInFlightTimeoutMs: Long = get(PUSH_LIMIT_IN_FLIGHT_TIMEOUT)
   def pushLimitInFlightSleepDeltaMs: Long = get(PUSH_LIMIT_IN_FLIGHT_SLEEP_INTERVAL)
   def pushSplitPartitionThreads: Int = get(PUSH_SPLIT_PARTITION_THREADS)
+  def partitionSplitMode: PartitionSplitMode = {
+    get(PARTITION_SPLIT_MODE) match {
+      case "hard" => PartitionSplitMode.HARD
+      case _ => PartitionSplitMode.SOFT
+    }
+  }
```

Comment:
Let's guard again to make the code more robust, in case someone changes the definition but forgets this method.
```suggestion
  def partitionSplitMode: PartitionSplitMode = {
    get(PARTITION_SPLIT_MODE) match {
      case "hard" => PartitionSplitMode.HARD
      case "soft" => PartitionSplitMode.SOFT
      case unknown => throw new IllegalArgumentException(balabala)
    }
  }
```